src/view/com/util/post-embeds/VideoEmbedInner/web-controls/VideoControls.tsx

```diff
@@ -413,7 +416,7 @@ export function Controls({
         <View
           pointerEvents="none"
           style={[a.absolute, a.inset_0, a.justify_center, a.align_center]}>
-          {buffering && <Loader fill={t.palette.white} size="lg" />}
+          {showSpinner && <Loader fill={t.palette.white} size="lg" />}
```

Comment:
should the above condition use `showSpinner` too instead of `buffering`? i have not tested visually closely